modules/utxo-lib/src/networks.ts

```diff
@@ -58,7 +58,7 @@ export type Network = {
     pubKeyHash: number;
     scriptHash: number;
   };
-  bech32?: string;
+  bech32: string;
```

Comment:
why do we need this?

Reply:
To match the Network type from `bitocinjs-lib` I'd love a better idea if you have one. I considered changing the upstream type in my "BitGo specific" commit, but it looked like it would become a maintenance pain.